test/scenarios.spec.ts

```diff
@@ -312,27 +329,36 @@ suite('== Custom Handler function scenarios ==', () => {
         expect(dateControlState.state.value).eq('2020-01-01');
     });
 
-    test('Check conflicts in canHandle throws a error log', async () => {
+    test('Check conflicts in canHandle throws an error', async () => {
         const rootControl = new DateSelectorManager().createControlTree();
         const input = TestInput.of(
             ValueControlIntent.of(AmazonBuiltInSlotType.DATE, {
                 'AMAZON.DATE': '2018',
                 action: $.Action.Set,
             }),
         );
-        const spy = sinon.stub(Logger.prototype, 'error');
+        const result = new ControlResultBuilder(undefined!);
+        try {
+            await rootControl.canHandle(input);
+        } catch (e) {
+            expect(e.message).eq(
+                'Matching custom and standard handler found in control: dateControl. Handlers in a single control should be mutually exclusive. handlers: ["SetValue","SetWithValue (built-in)"].One solution is to set the standardOverrides prop in the customHandlerFunc to override the standard built in handler.',
+            );
+        }
+    });
+
+    test('Check conflicts overrides on customHandlingFuncs in canHandle', async () => {
```

Comment:
how are you checking if conflicts are overridden by customHandlingFuncs here in this test case?